app/views/graphiql/rails/editors/show.html.erb

```diff
@@ -4,6 +4,9 @@
     <title>GraphiQL</title>
     <%= stylesheet_link_tag("graphiql/rails/application") %>
     <%= javascript_include_tag("graphiql/rails/application") %>
+    <% if GraphiQL::Rails.config.csrf %>
+    <%= csrf_meta_tags %>
```

Comment:
Do we still need the meta tag if we're sending the token by request header? (I _thought_ the headers were just so JS could add it to request headers)